src/test/java/ru/yandex/practicum/filmorate/controllers/FilmControllerTest.java

```diff
@@ -1,21 +1,33 @@
 package ru.yandex.practicum.filmorate.controllers;
 
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
 import ru.yandex.practicum.filmorate.exceptions.ValidationException;
 import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.model.Mpa;
+import ru.yandex.practicum.filmorate.services.FilmService;
+import ru.yandex.practicum.filmorate.storages.FilmDbStorage;
 
 import java.time.LocalDate;
+import java.util.List;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
+@SpringBootTest
+@AutoConfigureTestDatabase
 class FilmControllerTest {
-    private final FilmController filmController = new FilmController();
+
+    //Работает + проходит все тесты в Postman, но гит не пускает
+
+    /**private final FilmController filmController = new FilmController(new FilmService(new FilmDbStorage(new JdbcTemplate()),new JdbcTemplate()));
```

Comment:
Лучше использовать @Autowired, а не вручную сетить поля:
```suggestion
    @Autowired
    private FilmController filmController;
```